src/test/groovy/org/gradle/profiler/AbstractProfilerIntegrationTest.groovy

```diff
@@ -25,8 +25,9 @@ abstract class AbstractProfilerIntegrationTest extends AbstractIntegrationTest {
         "3.3", "3.5",
         "4.0", "4.7",
         "5.2.1", "5.6.3",
-        "6.0.1", "6.1", "6.6.1",
-        "7.1.1", "7.5"
+        "6.0.1", "6.9.4",
+        "7.1.1", "7.6.4",
+        "8.10.2"
```

Comment:
❓ Does adding to this list increase the time it takes to test the profiler?

I wonder what the use of testing with Gradle 3.3 is. Profiler is for people conscious about performance; using old Gradle versions or running on old versions of Java should be unimportant for them.

For example, for Gradle I think supporting the latest two major versions should be far more than enough.

Also, if we want to add the same level of support for `8.x`, we should include `8.0.2`. Or we can drop this notion, and from each major we could include only the latest minor/patch release.

Reply:
I removed one and added one, so I think it should be similar. :)

> I wonder what the use of testing with Gradle 3.3 is. Profiler is for people conscious about performance; using old 
Gradle versions or running on old versions of Java should be unimportant for them.

Good point. I think testing older versions is only useful for version comparison tests.

I agree we could remove some versions. What if we include just latest version for every major. And for latest major we test first and last version? We can probably also get rid of 3.x, 4.x, 5.x. So maybe: 
```
6.9.4, 7.6.4, 8.0.2, 8.10.2
```
Wdyt? 

Or should we also keep maybe 5.x, 4.x?

Reply:
Note: with https://github.com/gradle/gradle-profiler/pull/580 I figured that with Java 17 we will only support 7.3+, unless we add support to gradle-profiler to start Gradle under test with different JDK

Reply:
Done, we now test just `6.9.4, 7.6.4, 8.0.2, 8.10.2`, let me know if we should also include some older major version

Reply:
I kept also 5.6.4, since some tests check if some logic is executed if run with Gradle < 6.0. We can clean that up later, but I don't want to spent to much time cleaning it right now